src/js/ProjectionEnum.js

```diff
@@ -29,29 +29,29 @@
  *****************************************************************************/
 export let ProjectionEnum = {
    // Zenithal
-   TAN: {id: 1, fov: 180},	  /* Gnomonic projection      */
-   STG: {id: 2, fov: 360},	  /* Stereographic projection */
-   SIN: {id: 3, fov: 180},	  /* Orthographic		         */
-   ZEA: {id: 4, fov: 360},	  /* Equal-area 		         */
-   FEYE: {id: 5, fov: 190},
-   AIR: {id: 6, fov: 360},
+   TAN: {id: 1, fov: 180, longName: "gnomonic"},	  /* Gnomonic projection      */
```

Comment:
I would have chosen maybe something like: 'name' or 'label' instead of 'longName'

Reply:
I was kinda un-inspired. "label" sounds perfect thanks :slightly_smiling_face:

Reply:
Yes, but then I'd need to rename the code into "code" because everywhere else it's "name" for us. Do we do that?

Reply:
ok, let's keep it as you update it.